test/powershell/Language/Scripting/Trap.Tests.ps1

```diff
@@ -0,0 +1,24 @@
+
+Describe "Test trap" -Tags "CI" {
+    Context "Line after exception should not be invoked" {
+        It "line after exception should not be invoked" {
+            $a = . {trap {"trapped"; continue;}; . {"hello"; throw "exception"; "world"}}
+            $a.Length | Should Be 2
```

Comment:
I think the test should be more explicit in the expected results so it's easier to see the expected flow and to avoid a cascade of failures from helping the test succeed when it should fail.